css/main.css

```diff
@@ -0,0 +1,100 @@
+/* Define standard variables and values for website */
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 70vh;
+  background-color: #ff4dff;
+}
+
+.page-header {
+  border: none;
+  margin-top: 25%;
+  size-adjust: 190%;
+  -webkit-text-size-adjust: 100%;
```

Comment:
What does this do and why do we have them here?

Reply:
Those stylings are for displaying the text content on a mobile browser.